.github/workflows/if-nodejs-release.yml

```diff
@@ -80,8 +80,13 @@ jobs:
       - if: steps.packagejson.outputs.exists == 'true'
         name: Install dependencies
         run: npm install
+      - if: steps.packagejson.outputs.exists == 'true'
+        name: Add plugin for conventional commits
+        run: npm install conventional-changelog-conventionalcommits@5.0.0
+        working-directory: ./.github
       - if: steps.packagejson.outputs.exists == 'true'
         name: Publish to any of NPM, Github, and Docker Hub
+        working-directory: ./.github
```

Comment:
My concern: if we run this script in the `./.github` directory, will it then see project in the root?

Reply:
yes, working dir is basically set to point where release config file is. It works like a charm with go projects -> https://github.com/asyncapi/parser-go/blob/master/.github/workflows/release.yml#L54 (I just changed location to move it out of `workflows` dir)

Reply:
@magicmatatjahu you are right! I assumed all is good but in go projects there is no package json.

I did local dry run and got `ENOPKG Missing `package.json` file.`

I made proper changes. There is no way to point to releaserc file with a flag, so we just have to move it to the root